doc/devel/development_setup.rst

```diff
@@ -60,6 +60,15 @@ true for ``*.py`` files.  If you change the C-extension source (which might
 also happen if you change branches) you will have to re-run
 ``python -m pip install -ve .``
 
+Installing pre-commit hooks
+===========================
+You can optionally install `pre-commit <https://pre-commit.com/>`_ hooks.
+These will automatically check flake8 and other style issues when you run
+`git commit`. To install the hooks ::
```

Comment:
```suggestion
``git commit``. To install the hooks ::
```

Reply:
Thanks!

Reply:
Probably should include instructions for where the yaml is in case folks need to tweak it?